src/main/java/com/harrison/reflections/repository/EmployeeReadRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.harrison.reflections.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.harrison.reflections.domain.EmployeeRead;
+import com.harrison.reflections.domain.EmployeeWrite;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.harrison.reflections.domain.EmployeeWrite'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)